locksmith/src/operations/userOperations.ts

```diff
@@ -0,0 +1,47 @@
+const ethJsUtil = require('ethereumjs-util')
+const models = require('../models')
+const { User, UserReference } = models
+import * as types from '../types'
```

Comment:
Since you only use `UserCreationInput` maube use 
```suggestion
import { UserCreationInput } as types from '../types'
```